### ISSUE TITLE
set default visualitation mode

### ISSUE THREAD

**Issue by carlbaubau** [rhiokim/haroopad#246]
Threre are few option to open a md file in preview mode only?

**Comment by rhiokim** [comment]
It's not yet.
How about using keyboard shortcuts?

Mac : Cmd+Option+4
Win and Linux : Ctrl+Alt+4

And I'll consider the options for this.

**Comment by carlbaubau** [comment]
Yes, I know but my scope with markdown is create a college of note
(principal informatics personal guide or tips) and read and search them in
each device e computer that I have.
I try evernote, springpad and Google drive in this  years.. but the power
and the beautiful of md  win over them.
The best solution previous md is google drive but it It is too slow to open
a document, not permit a good offline use and for source snippets I a must
create table 1x1 and set color and font each time.

For this motivation a started to use md. I'm falling in love for you app
but eath time that I open a file only for read somethink (90% of case) I
must use shortcuts for only view. Other problem is that your app reopen
every doc that a use last time.

I know that is only my particalar use of md that make this problematics but
I think that if is not too expensive in time and work maybe you will
implement on one of next release.

Other idea that I give you is: evernote like program based on md and
dropbox storage?
I think that a day if I have the time maybe try to develop same thing of it
but you are 200 year in front to me to do it :-)

Thanks for all
Best regards
Alessandro
Il 27/nov/2013 05:59 "Rhio Kim" notifications@github.com ha scritto:

**Comment by MareoRaft** [comment]
@rhiokim Would really like a default visualization mode setting.  When I open a new document, I want it to be in "Editor" mode.  It's a dealbreaker if I need to change the mode every time I open a document.

Thanks for this creative and cool software!
